common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -39,7 +38,7 @@
  * 3.Write a JCA provider that registers the factory
  */
 public class PlainSaslServer implements SaslServer {
-  private String mAuthcid;
+  private String mAuthzid;
```

Comment:
Also, does that mean this variable is not currently used but for future authorization usage? If that is the case, please also javadoc that.